native/src/routes/categories/containers/CategoriesContainer.js

```diff
@@ -101,10 +123,13 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType): StatePropsTy
   }
 
   const resourceCacheUrl = state.resourceCacheUrl
-  if (resourceCacheUrl === null || state.cities.status === 'loading' || switchingLanguage ||
-    (route.status === 'loading' && !routeHasOldContent(route)) || languages.status === 'loading') {
+  const { models, children, allAvailableLanguages } = route
+  if (resourceCacheUrl === null || state.cities.status === 'loading' || languages.status === 'loading' ||
+    (route.status === 'loading' && (!models || !allAvailableLanguages || !children))) {
     return { status: 'loading', progress: resourceCache.progress }
   }
+  // $FlowFixMe Flow does not get that models and children cannot be undefined as it is already checked above
+  const stateView = new CategoriesRouteStateView(route.path, route.models, route.children)
```

Comment:
```suggestion
  const stateView = new CategoriesRouteStateView(route.path, models, children)
```

Using this should fix the flowfixme for models and children.

Reply:
Sadly not :/

Reply:
Maybe you also need to check route.path above.

Reply:
No, path is not the problem